scripts/utils.py

```diff
@@ -0,0 +1,71 @@
+import cv2
+import ffmpeg
+
+"""
+Miscellaneous utility functions that apply to multiple scripts.
+"""
+
+
+"""
+check_rotation and correct_rotation adapted from 
+https://stackoverflow.com/questions/53097092/frame-from-video-is-upside-down-after-extracting
+to handle the fact that some videos store rotation metadata while others do not,
+and OpenCV can't tell the difference.
+"""
+
+
+def check_rotation(path_video_file: str):
+    # only .mov files need to be rotated
+    if path_video_file.split('.')[-1] != '.MOV' or '.mov':
```

Comment:
this line is wrong. please change to `if path_video_file.split('.')[-1].upper() != '.MOV':`